multiversx_sdk_cli/projects/report/do_report.py

```diff
@@ -22,14 +25,25 @@ def do_report(args: Any, build_options: Any) -> None:
 
 
 def _build_report(args: Any, build_options: Dict[str, Any]) -> None:
-    base_path = Path(args.project)
+    base_path = Path(args.path)
     project_paths = get_project_paths_recursively(base_path)
     options = get_default_report_features()
-    report_creator = ReportCreator(options, skip_build=args.skip_build, skip_twiggy=args.skip_twiggy, build_options=build_options)
+
+    args_copy = copy.deepcopy(args)
```

Comment:
Can we move this within `_prepare_args_for_build` (thus, the preparation of args would be a bit more encapsulated - that is, the copying of the args is bit of a lower-level detail)? And `_prepare_args_for_build()` would actually return the new, copied & cleaned (filtered) object.

Reply:
Done!